docs/pages/management/dynamic-resources/teleport-operator.mdx

```diff
@@ -28,6 +28,7 @@ could cause instability and non-deterministic behaviour.
 Currently supported Teleport resources are:
 - users
 - roles
+- provision tokens
```

Comment:
Good to see this addition, but a simple example would be helpful.

Reply:
Filed https://github.com/gravitational/teleport/issues/37715 as this PR is large enough.

This will be addressed this quarter.